pif_ingestor/globus.py

```diff
@@ -0,0 +1,84 @@
+import os
+from globus_sdk import GlobusError, TransferData
+from mdf_forge import toolbox
+
+interval_time = 10
+default_collection = 35  # MDF Test collection
+
+def push_to_globus(paths, metadata={}, collection=default_collection, src_ep=None, verbose=False):
```

Comment:
Why specify a default collection?

Reply:
Mostly ease of testing. Also because the collection for Publish has to be its ID number, which is a little opaque and not always trivial to obtain. In real use, the collection will almost always be the MDF collection, I think.